.ci/install_crio.sh

```diff
@@ -113,6 +112,16 @@ echo "Installing CRI Tools"
 crictl_url="${crictl_repo}/releases/download/v${crictl_version}/crictl-${crictl_tag_prefix}${crictl_version}-linux-$(${cidir}/kata-arch.sh -g).tar.gz"
 curl -Ls "$crictl_url" | sudo tar xfz - -C /usr/local/bin
 
+# Change CRI-O configuration options
+crio_config_file="/etc/crio/crio.conf"
+
+# This commit contains changes regarding CRI-O’s default configuration files,
+# so we have to use a new default config path from here.
+# https://github.com/cri-o/cri-o/commit/0f1226b99685f95e83c94dc6668b6452df5056db
+if git merge-base --is-ancestor 0f1226b99685f95e83c94dc6668b6452df5056db HEAD; then
```

Comment:
nit: I wonder if we'd like to re-direct the git output to `/dev/null` or somewhere, for in the case where the patch fails, we will probably get a big 'git fail' error in the logs, which might be confusing when checking unrelated CI failures. Not a biggie though.

Reply:
Hm the git command does not output anything in success and error case :man_shrugging:

Reply:
> Hm the git command does not output anything in success and error case 🤷‍♂

Oh, OK - cool. surprised about the error case ;-).